src/components/Progress/ASABProgress.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Progress } from 'reactstrap';
+
+import './ASABProgress.scss';
+
+export function ASABProgress({
+	showLabel = true,
+	children,
+	...props
+}) {
+	// Handle both numeric and string values;
+	let progressValue;
```

Comment:
You can already declare progressValue here as `let progressValue = props.value;` and then use it e.g. in variables as `isNumeric`, etc..that way you would not have to retrieve `props.value` several times and repeat yourself